test/common/3scale_smtp.go

```diff
@@ -477,6 +493,27 @@ func sendTestEmail(ctx *TestingContext, t TestingTB) {
 	}
 }
 
+func rest3ScaleSecret(ctx *TestingContext, t TestingTB) (string, error) {
```

Comment:
```suggestion
func reset3ScaleSecret(ctx *TestingContext, t TestingTB) (string, error) {
```

Reply:
is there an extension to IDE to check for spelling? :facepalm: Not a single PR without a typo. huh?